src/box/sql/mem.c

```diff
@@ -3773,7 +3773,7 @@ port_c_get_vdbemem(struct port *base, uint32_t *size)
 		}
 		i++;
 	}
-	return (struct sql_value *) val;
+	return (struct Mem *) val;
```

Comment:
`val` is already of type `struct Mem *`, so you don't need a cast here.

Reply:
Updated:

```diff
@@ -3621,7 +3621,7 @@ port_lua_get_vdbemem(struct port *base, uint32_t *size)
                        goto error;
                }
        }
-       return (struct Mem *)val;
+       return val;
 error:
        for (int i = 0; i < argc; i++)
                mem_destroy(&val[i]);

```

Reply:
I don't see this change.

Reply:
Yes, but for `port_c_get_vdbemem` I've removed casting too.